config/samples/xgboost-dist/README.md

```diff
@@ -248,3 +249,242 @@ Events:
   Normal  ExitedWithCode           38s                xgboostjob-operator  Pod: default.xgboost-dist-iris-test-predict-master-0 exited with code 0
   Normal  XGBoostJobSucceeded      38s                xgboostjob-operator  XGBoostJob xgboost-dist-iris-test-predict is successfully completed.
 ```
+
+**Start the distributed XGBoost train to store the model locally**
+Before proceeding with training we will create a PVC to store the model trained.
+Creating pvc : 
+create a yaml file with the below content 
+pvc.yaml
+```
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: xgboostlocal
+spec:
+  storageClassName: glusterfs
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 10Gi
+```
+```
+kubectl create -f pvc.yaml
+```
+Note: Please use the storage class which supports ReadWriteMany i have used the glusterfs.
```

Comment:
Instead of "i have used",
Please use the storage class which supports ReadWriteMany. The example yaml above uses glusterfs.

Reply:
done